src/components/PersonInfo/PersonInfo.tsx

```diff
@@ -0,0 +1,58 @@
+import { useParams } from 'react-router-dom';
+import React from 'react';
+import classNames from 'classnames';
+import { Person } from '../../types/Person';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+interface Props {
+  person: Person;
+  mother?: Person;
+  father?: Person;
+}
+
+export const PersonInfo: React.FC<Props> = ({ person }) => {
```

Comment:
Why do you need mother and fother in the Props interface if you don't use them?

Reply:
Done! Thank you for your review!